### DISCUSSION TITLE
[教程]如何配置简悦的资料库功能

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2956] [Developer]
# 重要提示

> ### 嗨，简悦全新的 [教程中心](https://www.yuque.com/kenshin/simpread) 已上线，**下面的内容并不一定是最新的**，建议 [访问这里](https://www.yuque.com/kenshin/simpread/vcmg4o) 查看最新版教程。

***

# 资料库

> 在 [知识库](https://github.com/Kenshin/simpread/discussions/2221) 的基础上，通过简悦，可以让你的**坚果云或 Dropbox 特定文件夹的内容在线查看**。

## 前提

> 此功能需要先配置 [知识库](https://github.com/Kenshin/simpread/discussions/2221)。

## 支持的同步盘

1. 坚果云
2. Dropbox

## 阅读列表

> 当 `output` 文件夹的内容为 HTML 时，可通过 [阅读列表](https://github.com/Kenshin/simpread/discussions/2954) 在线查看，支持：

- PWA，包括：桌面端 · Android 端

- iOS 端

- 任意设备的浏览器在线查看

## 注意

1. 资料库功能需要配置自动同步，且仅限：坚果云与 Dropbox，只能是 `<网盘根目录>/SimpRead/output` 目录结构，不可更改。

2. 如果单纯的使用增强导出的话，可使用任意位置的目录，同时也可以不用配置自动同步功能。

3. 如果只是使用自动同步的话，也可不限任意网盘，如：iCloud 均可。

# 引申

> 为了让加入到 `output` 里面的内容更加的自动化，比如下面的流程，操作后会自动加入到此文件夹：

- 当加入稍后读后
- 当产生标注时
- 使用导出到本地 HTML or 离线 HTML 时

需要配置 **增强导出与自动化** 功能。

## 配置增强导出

> 细节 [如何配置增强导出](https://github.com/Kenshin/simpread/discussions/2958)。

## 配置自动化

> 通过配置 [自动化](http://ksria.com/simpread/docs/#/自动化) 功能，可方便的在简悦的阅读模式中将任意内容导入到 `output` 文件夹中。

## 关联

- [如何让稍后读支持读取本地缓存 #2146](https://github.com/Kenshin/simpread/discussions/2146)
- [当产生标注时，如何自动同步到本地 #2220](https://github.com/Kenshin/simpread/discussions/2220)
- [当产生标注时，如何同时下载标注与原文 #2147](https://github.com/Kenshin/simpread/discussions/2147)
- [让简悦变成你的知识库（离线且具有永久链接） #2221](https://github.com/Kenshin/simpread/discussions/2221)
- [将自动化服务变成你的导出服务 #2151](https://github.com/Kenshin/simpread/discussions/2151)

**Comment by chinyajie** [comment]
请问我访问阅读列表时显示“获取失败，请确保 output 文件夹的存在”。我的坚果云对应的目录不是上面提到的 <网盘根目录>/SimpRead/output；而是 <网盘根目录>/SimpRead/md 。我该如何修改，我没找到修改的办法，也不确信是否是这个问题，辛苦帮助看看。

**Reply by Kenshin** [reply]
嗨，你可以这样做：

1. 确保 SimpRead 在网盘根目录下。
2. 没有 output 的话，新建它。

**Reply by wanghusw** [reply]
你好，坚果云的 <网盘根目录> 一定得是 `我的坚果云` 吗？建立的其他和它同级的文件夹是不是不可以？
![Snipaste_2021-12-27_14-24-15](https://user-images.githubusercontent.com/26402076/147441273-98e9ac50-d7b7-45a7-a191-05e21074c419.png)
？

**Reply by Kenshin** [reply]
@wanghusw 

网盘根目录应该不是我的坚果云，而是跟它同级目录。

**Reply by wanghusw** [reply]
我是不是可以这样理解，我有一个 `SimpRead` 文件夹，我要建立 `output` 文件夹的话，路径是 `SimpRead/SimpRead/output`？

**Reply by Kenshin** [reply]
> 我是不是可以这样理解，我有一个 `SimpRead` 文件夹，我要建立 `output` 文件夹的话，路径是 `SimpRead/SimpRead/output`？

当然不是了，你可以看 https://www.yuque.com/kenshin/simpread/bwcd6n 里面的描述，即下图所示

![image](https://user-images.githubusercontent.com/81074/147441972-f7278089-3053-4973-a2d8-2dc3538927ed.png)

正确的文件夹应该是  `<网盘>/SimpRead/output`
